libs/blocks/graybox/graybox.js

```diff
@@ -264,7 +264,16 @@ const setupChangedEls = (globalNoClick) => {
   }
 };
 
+const transformLinks = () => {
+  const grayboxHostname = window.location.hostname;
+  const consumerHostname = grayboxHostname.replace(/[^.]+\.(?:([^.]+)-)?graybox\./, (_, sub) => `${sub || 'www'}.`);
+  document.querySelectorAll(`a[href*="${consumerHostname}"]`).forEach((el) => {
+    el.href = el.href.replace(consumerHostname, grayboxHostname);
+  });
+};
+
 const grayboxThePage = (grayboxEl, grayboxMenuOff) => () => {
+  transformLinks();
```

Comment:
What happens to links loaded later on, such as the ones in the footer, for example? Does this logic apply to them as well, since those links are loaded with a delay?

Reply:
Good catch 👍 The footer links will not be converted because of the loading delay. This is not an issue currently since navigations are grayed out by default.
I will create a new ticket for handling async sections, as this has more implications than just the links conversion, so it falls out of the scope of this ticket.